src/components/TokenLocking/MilesReceipt.tsx

```diff
@@ -25,44 +28,57 @@ const MilesReceipt = ({
 
   return (
     <Modal open={open} onClose={onClose} slotProps={{ backdrop: { sx: { backgroundColor: '#636669BF' } } }}>
-      <Box display="flex" flexDirection="column" alignContent="center">
-        <Grid container direction="row" className={css.milesReceipt}>
-          <div className={clsx(css.gradientBackground, { [css.unlockGradient]: isUnlock })} />
-
-          <Grid item xs={12} md={8}>
-            <Stack className={css.leftReceipt}>
-              <SvgIcon component={SafeExplorers} inheritViewBox sx={{ width: '282px', height: 'auto' }} />
-              <Stack mt="auto">
-                <Typography variant="h1" component="div" fontWeight="bold" mb={1}>
-                  {isUnlock ? 'Your tokens are unlocked' : 'Welcome on board'}
-                </Typography>
-                <Typography mb={5}>
-                  You successfully started {isUnlock ? 'unlocking' : 'locking'} {amount} SAFE. Once the transaction is
-                  signed and executed the changes will be reflected in this App.
-                  <br />
-                  {isUnlock ? 'Withdrawal will be available in 24h' : ''}
-                </Typography>
+      <>
+        <div className={clsx(css.gradientBackground, { [css.unlockGradient]: isUnlock })} />
+        <Stack className={css.milesReceipt}>
+          <SvgIcon component={SafePass} inheritViewBox sx={{ width: '130px', height: 'auto' }} />
+          <Stack mt="auto" className={css.topReceipt}>
+            <Typography variant="h1" component="div" fontWeight="bold" mb={1} mt={7}>
+              {isUnlock ? 'Unclocking' : 'Locking'} started...
+            </Typography>
+            <Typography mb={4}>
+              You successfully started {isUnlock ? 'unlocking' : 'locking'} {amount} SAFE. You successfully started{' '}
+              {isUnlock ? 'unlocking' : 'locking'} {amount} SAFE. Once the transaction is signed and executed the
+              changes will be reflected in this App.
+              <br />
+              {isUnlock ? 'Withdrawal will be available in 24h' : ''}
+            </Typography>
 
-                {!isUnlock && (
-                  <Stack direction="row" gap={2}>
-                    {/* TODO: Need content to share message on Twitter */}
-                    <Button variant="outlined" href="https://twitter.com/safe" target="_blank">
-                      Share on X
-                    </Button>
-                    <Button variant="outlined" href="https://twitter.com/safe" target="_blank">
-                      Follow Safe on X
-                    </Button>
-                  </Stack>
-                )}
+            {!isUnlock && (
+              <Stack direction="row" gap={2} mb={2}>
+                <Button
+                  variant="outlined"
+                  href={`http://twitter.com/intent/tweet?&text=${TWEET_CONTENT}`}
+                  target="_blank"
```

Comment:
I would also set `rel="noreferrer" `for all external links. We could even extract this into some `ExternalLink` component with variants `Link` and `Button`. Considering that we will have to add multiple more external links in this app.